suite_cam6.xml

```diff
@@ -242,7 +242,7 @@
       <!-- **************************** -->
       <!-- Negative moisture/tracer check -->
       <!-- **************************** -->
-      <scheme>qneg4</scheme>                          <!-- Modify surface moisture and heat fluxes to prevent negative moisture values, and output diagnostics -->
+<!--      <scheme>qneg4</scheme>                         --><!-- Modify surface moisture and heat fluxes to prevent negative moisture values, and output diagnostics -->
```

Comment:
I might leave this uncommented, but rename `qneg4` to `qneg_surface`, with a comment noting that in CAM this is referred to as "qneg4".  The reason for this change is because every uncommented scheme in this file indicates a physics or chemistry process that must exist in some form as a CCPP scheme in order to exactly replicate CAM6 physics, and my fear is that if we comment it out my future self might think that it is un-necessary.

Reply:
I'm on board - changed to qneg_surface and uncommented.

Reply:
I think that `qneg_surface` should be its own scheme (in its own module). I agree that it should be uncommented here but maybe open an issue to remind us that it needs to be implemented.

Reply:
issue opened.